server/auth/authApi.js

```diff
@@ -1,21 +1,26 @@
 const passport = require('passport')
+const R = require('ramda')
 
 const {sendOk} = require('../serverUtils/response')
 
 const {userPrefNames, getUserPrefSurveyId} = require('./../user/userPrefs')
 const {getSurveyById} = require('../survey/surveyRepository')
 const {deleteUserPref} = require('../user/userRepository')
+const {validateSurvey} = require('../survey/surveyValidator')
 
-const sendResponse = (res, user, survey = null) => res.json({user, survey})
+const sendResponse = (res, user, survey = null, surveyValidation = null) => res.json({user, survey, surveyValidation})
```

Comment:
i don't think you are passing the validation as parameter to the function. isn't it a property of the survey?

Reply:
right...